src/routes.js

```diff
@@ -2,32 +2,35 @@ import React from 'react';
 import {Router, IndexRoute, Route, hashHistory} from 'react-router';
 import LoginForm from './containers/Auth/LoginForm';
 import SignupForm from './containers/Auth/SignupForm';
-import SignupSuccess from './components/SignupSuccess';
+import SignupSuccess from './components/MessagePage/signupSuccess';
+import PasswordReset from './containers/Auth/PasswordReset';
+import PasswordResetPending from './components/MessagePage/passwordResetPending';
 import ApiDocs from './components/ApiDocs';
 import {
-    App,
-    Auth,
-    Billing,
-    EditBucket,
-    NewBucket,
-    Buckets,
-    Dashboard,
-    FileBucket,
-    Home,
-    NotFound,
-    Support
-  } from 'containers';
+  App,
+  Auth,
+  EditBucket,
+  NewBucket,
+  Buckets,
+  Dashboard,
+  FileBucket,
+  NotFound,
+  Support
+  // Billing,
```

Comment:
Can remove these non-existent routes entirely; feel free to remove any stale references.